### PR TITLE
Update to newer QPU

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Minor updates to the Summary section. 
Note that the latest version seems to add ``attachments`` metadata: I opened in the IDE and it doesn't cause a problem but I can manually delete if anyone is worried.

### CODE REVIEW

**Comment by JoelPasvolsky** [comment]
The circleci test failure is due to "API token not defined", running on my PR and I think it used to run only on master, so I think it's just not getting the usual access to a token.

**Review by mdecandia — APPROVED** [review]
LGTM

File: 01-structural-imbalance-overview.ipynb [hunk 1]
```diff
@@ -379,22 +397,27 @@
...
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For a submission on February 13, 2020, the minor-embedding for the entire saved database (238 variables) required about half a D-Wave 2000Q QPU's qubits. Ocean software's [problem inspector](https://docs.ocean.dwavesys.com/en/stable/docs_inspector/sdk_index.html) tool shows the embedded problem: \n",
+    "For one particular submission to an Advantage quantum computer, the minor-embedding for the entire saved database (238 variables) represented one variable with a dozen qubits coupled strongly together (a [chain](https://docs.ocean.dwavesys.com/en/stable/concepts/embedding.html) of qubits). Chains are used when minor-embedding any but the smallest problems; however, such long chains can be unreliable and affect performance. \n",
+    "\n",
+    "Ocean software's [problem inspector](https://docs.ocean.dwavesys.com/en/stable/docs_inspector/sdk_index.html) tool shows the embedded problem, highlighting the long chain: \n",
     "\n",
     "<img src=\"images/EmbeddingFullDB_inspector.png\" width=700x/>"
```
Comment (randomir) [inline 01-structural-imbalance-overview.ipynb:408]: Did you mean to use `px` for length unit? I don think `x` is a valid [CSS unit](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#numbers_lengths_and_percentages).
```suggestion
    "<img src=\"images/EmbeddingFullDB_inspector.png\" width=\"700px\"/>"
```

**Review by randomir — APPROVED** [review]

File: 01-structural-imbalance-overview.ipynb (hunk 1 above)
Comment (JoelPasvolsky) [reply 01-structural-imbalance-overview.ipynb:408]: It looks like JNs accept `width=<number><anything goes>` here but the [docs](https://jupyterbook.org/en/stable/content/figures.html) do say `px`.

**Pull request merged**